src/redux/books/Books.js

```diff
@@ -1,31 +1,63 @@
 const ADD_BOOK = 'bookstore/src/redux/books/addBook';
 const REMOVE_BOOK = 'bookstore/src/redux/books/removeBook';
 
-export const addBook = () => (
+export const defaultState = [
+  {
+    id: '0a33d1ab',
+    title: 'The Hunger Games',
+    author: 'Suzanne Collins',
+    completed: '43%',
+    chapter: '14',
+  },
+  {
+    id: '1b35d1ab',
+    title: 'Dune',
+    author: 'Frank Herbert',
+    completed: '10%',
+    chapter: '1',
+  },
+  {
+    id: '0c36d3ab',
+    title: 'Capital in the Twenty-First Century',
+    author: 'Suzanne Collins',
+    completed: '93%',
+    chapter: '20',
+  },
+];
+
+export const addBook = (payload) => (
```

Comment:
- [ ] There is a problem happing because of this module name, here you name it `Books.js` with capital `B` and you imported it into your components as `book` and that's making compiling problems, So please fix that.

![image](https://user-images.githubusercontent.com/68796232/201991008-8190f171-83d9-4d6e-849e-30aff48cd49a.png)